onadata/apps/api/viewsets/data_viewset.py

```diff
@@ -515,7 +589,8 @@ def validation_status(self, request, *args, **kwargs):
 
         return Response(data, status=http_status)
 
-    @detail_route(methods=['GET', 'POST', 'DELETE'], extra_lookup_fields=['label', ])
+    @detail_route(methods=['GET', 'POST', 'DELETE'],
+                  extra_lookup_fields=['label', ])
```

Comment:
I realize this is just a whitespace change to code from 2013 or earlier, but the trailing comma is only needed for tuples, right?

Reply:
Right!